$OpenBSD: patch-configure2,v 1.1 2012/05/03 06:51:07 ajacoutot Exp $

Fix the big endian check to actually define WORDS_BIGENDIAN in the config.h header.

--- configure2.orig	Mon Apr 30 01:24:09 2012
+++ configure2	Mon Apr 30 01:25:07 2012
@@ -149,7 +149,12 @@ cat > config.h << EOF
 /* Automatically generated by configure, do not edit */
 #include "version.h"
 EOF
-test "$bigendian" = "yes" && echo "#define WORDS_BIGENDIAN" || echo "#undef WORDS_BIGENDIAN" >> config.h
+if test "$bigendian" = "yes"
+then
+    echo "#define WORDS_BIGENDIAN" >> config.h
+else
+    echo "#undef WORDS_BIGENDIAN" >> config.h
+fi
 
 # build tree in object directory if source path is different from current one
 if test "$source_path_used" != "no"; then
